miso-web/src/main/webapp/scripts/hot_sample.js

```diff
@@ -299,9 +303,35 @@ HotTarget.sample = (function() {
               flat.externalName = Utils.valOrNull(sam.externalName);
             },
             pack: function(sam, flat, errorHandler) {
+              sam.externalNames = flat.externalName;
               if (!getSelectedIdentity(flat)) {
                 sam.externalName = flat.externalName;
               } // else externalName will come from an existing Identity via the Identity Alias column
+            },
+            depends: 'identityAlias',
+            update:function(sam, flat, flatProperty, value, setReadOnly, setOptions, setData) {
+              if(!value || value == "Delete external name, select a project, then re-enter external name." || value == '(...searching...)'
+                || /First Receipt \(.*\)/.test(value) || !flat['externalName']){
+                setOptions({
+                  'renderer': Handsontable.renderers.TextRenderer
+                });
+                return;
+              };
+              
+              var existingExternalNames = getExternalNames(value);
+              var newExternalNames = flat['externalName'].toLowerCase().split(",").map(function(name){
+                return name.trim();
+              });
+              if(newExternalNames.every(function(externalName){
+                return existingExternalNames.indexOf(externalName) != -1;
+              })){
+                renderer = Handsontable.renderers.TextRenderer;
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Add the "let", "const" or "var" keyword to this declaration of "renderer" to make it explicit. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS2703)